demo/admin/src/products/ProductForm.tsx

```diff
@@ -262,6 +277,29 @@ function ProductForm({ id }: FormProps): React.ReactElement {
                         <Field name="image" isEqual={isEqual}>
                             {createFinalFormBlock(rootBlocks.image)}
                         </Field>
+                        <FormSection title={<FormattedMessage id="product.packageDimensions" defaultMessage="Package Dimensions" />}>
```

Comment:
maybe that should be a Fieldset, but that is not the important thing here

Reply:
We're adding the new FieldSet in #1082